.gitattributes

```diff
@@ -1,2 +1,4 @@
 # Do not do any text (line ending) normalization
-* -text
\ No newline at end of file
+* -text
+# Generate diff of .txt files as text file instead of binary file
+*.txt diff
```

Comment:
This should happen by default I think? Was the cause for being treated as a binary the fact that it was in the /bin folder? or was the encoding UTF-16 or something?

Also do we intend to move the location of the requirements files?

Reply:
It could be due to being ignored. Because I changed the encoding to UTF-8. About moving the requirements file - I was testing if it shows the diff if requirements is moved out of bin and yes it does show diff properly as text.

Reply:
Even if I excluded top level `bin` folder from gitgnore, it still shows up as binary :(

Reply:
Other files under `bin` are not treated as binary. I expect being UTF-16 originally is the cause. 
I would try removing the file, forcing that with `git rm -f` and then re-adding it as utf-8 in a new commit so that it's a new file.